articles/hosted-pages/login/index.md

```diff
@@ -0,0 +1,127 @@
+
+### Hosted Login vs Embedded Login
+
+Why use the Hosted Login Page rather than embedding your login functionality within your application?
+
+Security is the primary reason, followed by ease of setup. Cross-origin authentication is inherently more dangerous, and more likely to be vulnerable to man-in-the-middle attacks. Using the Hosted Login Page for the authentication process with Auth0 prevents that from ever being a concern. Additionally, the Hosted Login Page is very easy to implement, especially if a custom UI is not required.
+
+### Passwordless on Native Platforms
+
+Currently, the Hosted Login Page is the _only_ way to use Passwordless authentication on Native platforms. So if your use case is a native iOS or Android application, for example, and you intend to implement Passwordless, you will need to use the Hosted Login Page.
+
+## How to Use the Hosted Login Page
+
+### 1. Enable the Hosted Login Page
+
+In the [Auth0 Dashboard](${manage_url}), you can enable a custom Hosted Login Page by navigating to [Hosted Pages](${manage_url}/#/login_page) and enabling the **Customize Login Page** toggle.
+
+![Hosted Login Page](/media/articles/hosted-pages/login.png)
+
+### 2. Choose a Technology
+
+In order to get started using the Hosted Login Page, you'll first want to choose the technology that you'd like to use to power it. Click one of the links below to get started.
+
+- [Lock](/hosted-pages/login/lock) - Lock is the simplest way to use the HLP. Lock is a pre-built, customizable login widget that will allow your users to quickly and easily login to your application.
+- [Lock Passwordless](/hosted-pages/login/lock-passwordless) - Lock Passwordless uses the same style of interface as Lock, but rather than offering identity providers as login options, will simply ask the user to enter an email or SMS number to begin a Passwordless authentication transaction.
+- [Auth0.js v8](/hosted-pages/login/auth0js/v8) - Auth0.js is the SDK used for interacting with the Auth0 authentication API. Primarily, you would use the SDK if you need to build your own custom login UI, or implement more complex functionality than simply allowing your users to login. Auth0.js v8 is the most up to date version of that SDK, and is fully capable of OIDC compliant transactions.
+- [Auth0.js v7](/hosted-pages/login/auth0js/v7) - Auth0.js is the SDK used for interacting with the Auth0 authentication API. Primarily, you would use the SDK if you need to build your own custom login UI, or implement more complex functionality than simply allowing your users to login. Version 7 is the older supported version, and is only for use by applications which are not yet ready to upgrade to version 8, or cannot be OIDC compatible.
```

Comment:
Maybe we leave the doc, but don't reference it here as an option. I'm good with that.